google/cloud/iam/CMakeLists.txt

```diff
@@ -169,5 +169,7 @@ install(
 foreach (dir IN LISTS service_dirs)
     if (BUILD_TESTING AND GOOGLE_CLOUD_CPP_ENABLE_CXX_EXCEPTIONS)
```

Comment:
nit: we should probably do the `if` outside, instead of calling `target_link_libraries` from within the `foreach` loop.